scripts/epa/airdata/criteria_gases/criteria_gases_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Unit tests for criteria_gases.py
+
+Usage: python3 criteria_gases_test.py
+'''
+import unittest, csv, os, tempfile
+from criteria_gases import create_csv, write_csv, write_tmcf
+
+module_dir_ = os.path.dirname(__file__)
+
+
+class TestCriteriaGasesTest(unittest.TestCase):
+
+    def test_write_csv(self):
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            with open(
+                    os.path.join(module_dir_, 'test_data/test_import_data.csv'),
```

Comment:
Could we trim the import data to no more than a few 100 lines or so? That way the test data will be more readable.... (as such it does not even load in the diffs)

Reply:
ahh sorry, good idea -- done.